google-cloud-bigtable/src/main/resources/changestream.json

```diff
@@ -411,13 +411,110 @@
               "low_watermark": "2022-07-01T00:00:00Z",
               "mutations": [
                 {
+                  "set_cell": {
+                    "family_name": "family",
+                    "column_qualifier": "0000000000000000",
+                    "timestamp_micros": 1000,
+                    "value": "dmFsdWUtVkFM"
+                  }
+                }
+              ]
+            }
+          },
+          "error": false
+        }
+      ]
+    },
+    {
+      "description": "1 chunked SetCell([{SC_chunk1}, {SC_chunk2}, {SC_chunk3}]->ChangeStreamMutation{SC})",
+      "api_responses": [
+        {
+          "data_change": {
+            "row_key": "0000000000000000",
+            "type": "USER",
+            "source_cluster_id": "source-cluster-id",
+            "commit_timestamp": "2022-07-01T00:00:00Z",
+            "tiebreaker": 100,
+            "chunks": [
+              {
+                "chunk_info": {
+                  "chunked_value_size": 1
```

Comment:
this should be 13 and everywhere else should be updated

Reply:
And maybe we should verify this value with the previous chunk as a sanity check

Reply:
Done.